_episodes/01-run-quit.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Running and Quitting"
+teaching:  5
+exercises: 0
+questions:
+- "How can I run Python programs?"
+objectives:
+- "Launch iPython."
+- "Run some simple python commands."
+- "Exit the iPython interpreter."
+keypoints:
+- "Use iPython for editing and running Python."
+- "Use the keyboard and mouse to select and edit cells."
+
+---
+## Python programs are plain text files.
+
+*   They have the `.py` extension to let everyone (including the operating system) 
+    know it is a Python program.
+    *   This is convention, not a requirement.
+
+
+## Start iPython
+
+*   The [Anaconda package manager][anaconda] is an automated way to install the iPython on your own machine.
```

Comment:
do we want information about installing ipython right away since it will be on CAE machines. Maybe this should actually be a note at the end of the setup page rather than here so that they don't think they need to do this during the first lesson.

Reply:
That sounds good. I'll move it to the end. Thanks!

- [ ] move notes on Anaconda and iPython install to the end of the intro lesson